### DISCUSSION TITLE
NSClient++ Python 3.3.0

### DISCUSSION THREAD

**Discussion by AllMighty-Acorn** [mickem/nscp#788] [Q&A]
When I installed NSClient+++, Python 3.3.0 also appeared as an installed program. Python has many known vulnerabilities, so I was wondering if there would be issues with updating to the latest version of Python (3.11.1) or if uninstalling Python entirely is possible (I'm assuming no for this one).

**Comment by mickem** [comment]
In the installer you can disable the python script module which should not install python. Also a new version will soon be out which will have an updated version of python
